src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/config/HmppsExternalUsersApiExceptionHandler.kt

```diff
@@ -200,6 +201,20 @@ class HmppsExternalUsersApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(UserLastGroupException::class)
+  fun handleAuthUserLastGroupException(e: UserLastGroupException): ResponseEntity<ErrorResponse> {
+    log.debug("Auth user group exception caught: {}", e.message)
```

Comment:
Should just be handleUserLastGroupException - and logging should say external user group